assets/js/components/performance/Table/scripts.jsx

```diff
@@ -73,17 +74,24 @@ class TabularView extends Component {
     }
 
     renderTableHead() {
```

Comment:
probably worth using a Set for this kind of things generally https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/Set